internal/worker/modelworkermanager/modelworkermanager.go

```diff
@@ -21,18 +21,13 @@ import (
 	"github.com/juju/juju/core/http"
 	corelogger "github.com/juju/juju/core/logger"
 	"github.com/juju/juju/core/model"
+	modelerrors "github.com/juju/juju/domain/model/errors"
 	"github.com/juju/juju/internal/pki"
 	"github.com/juju/juju/internal/services"
 	internalworker "github.com/juju/juju/internal/worker"
 	"github.com/juju/juju/state"
 )
 
-// ModelWatcher provides an interface for watching the additiona and
```

Comment:
Also need to delete the interfaces below
- Controller
- Model
- RecordLogger

Reply:
Sure will do, removed type ControllerConfigGetter interface as well from the file as it isn't used, thanks!